dataframe/dataframe.go

```diff
@@ -255,17 +257,20 @@ func (df DataFrame) Set(indexes series.Indexes, newvalues DataFrame) DataFrame {
 	if df.Err != nil {
 		return df
 	}
+	if indexes == nil {
+		return DataFrame{Err: createErr("nil indexes", "df.Set()")}
```

Comment:
For chaining operations is a really useful feature, so you know the place where it first failed